ansible/roles/docker/README.adoc

```diff
@@ -1,7 +1,51 @@
-== Role: Docker
+= Role: Docker
+:toc:left
+:doctype: article
+:source-highlighter: rouge
 
-Cluster target type: K8s
 
-=== Description
+
+Installs docker on host.
+
+== Description
+
+This role will install a Docker on a linux machine.
```

Comment:
This is important to describe which pieces about docker we are installing; client, daemon, ....

Reply:
improved the description